scripts/easy-secrets-gen.sh

```diff
@@ -2,7 +2,22 @@
 
 cd $(git rev-parse --show-toplevel)
 
-DEST_DIR=${1:-.}
+DEST_DIR=${1:-components/01-secrets/}
```

Comment:
Why the path change here?

Reply:
That's where all the docs were pointing.

Reply:
`DEST_DIR` looks like it was being used as a temporary location when secrets were created, and then once they're sealed they are then additionally written to `components/01-secrets/`. Since the new method is not writing them prior to being sealed, and using `DEST_DIR` only to write the sealed secrets, this looks change valid to me.

Reply:
looking at this a bit more, I think what was there was probably preferred. if easy-secrets-gen is called without an arg, we'll assume writes to pwd. I believe 01-secrets is no longer being used.

Reply:
I was actually just about to follow up and say that `components/01-secrets` should be retired. I believe it's in the original doc which is why it hasn't been.

Reply:
The two secrets scripts need to get merged ultimately.